.github/workflows/tests_sources.yml

```diff
@@ -40,9 +40,9 @@ jobs:
           - os: ubuntu-20.04
             DEPENDENCIES_INSTALLATION: "sudo apt -y install clang-format-10 cppcheck"
```

Comment:
It seems we are not enforcing cppcheck version 2.10 on Linux, is that on purpose?

Reply:
I don't object to trying to do that but I'm not running Ubuntu so wasn't immediately familiar with how to do it and also it seemed unlikely that Ubuntu would upgrade the cppcheck version between Ubuntu releases.